x-pack/plugins/infra/public/components/logging/log_analysis_results/analyze_in_ml_button.tsx

```diff
@@ -62,7 +62,7 @@ const getOverallAnomalyExplorerLink = (pathname: string, jobId: string, timeRang
     },
   });
 
-  const hash = `/explorer?${stringify(urlUtils.encodeQuery({ _g }), { encode: false })}`;
+  const hash = `/explorer?${stringify(urlUtils.encodeQuery({ _g }))}`;
```

Comment:
I think the above would be fixed with:

```suggestion
  const hash = `/explorer?${makeUrlFromQuery({ _g }}`;
```